lib/omnibus/packagers/base.rb

```diff
@@ -177,11 +177,11 @@ def skip_packager(val = false)
     #   - setup
     #   - build
     #
-    def run!
+    def run!(duration_cb = nil)
```

Comment:
```suggestion
    def run!
```
I'd rather avoid `run!` from having this sort of side-function; see next comment for idea.